gcloud/datastore/query.py

```diff
@@ -418,7 +418,10 @@ def next_page(self):
         #       for discussion.
         entity_pbs, cursor_as_bytes, more_results_enum = query_results[:3]
 
-        self._start_cursor = base64.b64encode(cursor_as_bytes)
+        if cursor_as_bytes == b'':
```

Comment:
I was imagining we'd fix this "at the source" (in `Connection.run_query`), but even doing that would've required a branch here.